worlds/animal_well/region_data.py

```diff
@@ -788,6 +986,8 @@ class AWData(NamedTuple):
     rname.frog_under_ostrich_statue: {
         rname.frog_near_wombat:  # may have to go a few screens away to hit a switch, but you don't need items
             AWData(AWType.region),
+        lname.fruit_92:
+            AWData(AWType.location, [[iname.can_break_spikes]], loc_type=LocType.fruit),
```

Comment:
```suggestion
            AWData(AWType.location, [[iname.disc], [iname.ball], [iname.yoyo], [iname.top]], loc_type=LocType.fruit),
```
Fix for the bug reported in discord